src/Queue/AWSSQS/SqsQueueManager.php

```diff
@@ -309,4 +322,13 @@ private function getPrefixedQueueName(string $queueName): string
 
         return $prefixedQueueName;
     }
+
+
+    public function terminateGracefully(): void
+    {
+        // todo - remove when tested
+        $this->logger->debug('SqsQueueManager::terminateGracefully() called');
+
+        $this->isBeingTerminated = true;
```

Comment:
todo - remove when tested
```suggestion
        $this->isBeingTerminated = true;
```